handler/datasource/pack.go

```diff
@@ -40,10 +40,10 @@ func packHandler(
 	db *gorm.DB,
 	ctx context.Context,
 	resolver datasource.HandlerResolver,
-	chunkID uint64,
+	chunkID uint32,
 ) ([]model.Car, error) {
 	var chunk model.Chunk
-	err := db.Where("id = ?", chunkID).Find(&chunk).Error
```

Comment:
Please manually load the Item. See #190  and https://github.com/data-preservation-programs/singularity/blob/7e8e8cccdd2f7df4fc773516215a8da4e900df47/handler/datasource/inspect/chunkdetail.go#L37
Preloadiing Source.Dataset and Preloading ItemParts are fine. But Items needs to be preloaded manually. Otherwise this will hit SQL max parameters.

Reply:
Worth putting this to model.Chunk's method